libs/langchain/langchain/retrievers/google_cloud_enterprise_search.py

```diff
@@ -71,6 +71,12 @@ class GoogleCloudEnterpriseSearchRetriever(BaseRetriever):
     when making API calls. If not provided, credentials will be ascertained from
     the environment."""
 
+    engine_data_type: int = Field(default=0, ge=0, le=1)
+    """ Defines the enterprise search data type
+    0 - Unstructured data 
+    1 - Structured data
```

Comment:
Should this be expanded to cover Website data as well?

And would it make sense to make this an enum?

Reply:
I will add a TODO - I was considering ENUM but found it more complicated for the user to pass that as the __init__ automates the assignment

Reply:
Ok, that makes sense.